spec/models/ability_spec.rb

```diff
@@ -408,7 +408,7 @@
     let(:team) { create(:team) }
 
     it 'should be logged in' do
```

Comment:
Hm, what is this test supposed to test in the end?

As said in a previous comment, I don't really think that abilities need to have something to do with being signed in or not in our case...

Reply:
I'm not sure I understand, sorry :(

Should I remove this test?

Reply:
Yes, I'd suggest to remove it 👍

Reply:
It's gone! Thanks @klappradla